addons/l10n_sa_edi/tests/compliance/simplified/credit.xml

```diff
@@ -0,0 +1,178 @@
+<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
```

Comment:
We could mock generate UBL files, but the files under tests/compliance are there for running compliance checks for ZATCA, not for unit tests. I believe it would be easier to just use pregenerated xml files, though if that's not an option, mock generation is doable

Reply:
Do we need to have these files in the code then?  

Anyways, there is too much code in this module not to have any unit tests. It would be good to test the XML generation in mocked form at least (so we don't depend on government servers to make pass the tests on runbot) and specially the part that is not UBL standard.

Reply:
I believe the unit tests are already being worked on, so it is safe to close this